test/specs/carousel.test.js

```diff
@@ -533,6 +533,42 @@ describe('<Carousel />', () => {
 
         expect(defaultWrapper).toHaveState({ cellAlign: 'left' });
       });
+
+      it('should move to next slide when pressing right arrow key', () => {
+        const map = {};
+        document.addEventListener = jest.fn((event, cb) => {
+          map[event] = cb;
+        });
+
+        const wrapper = mount(
+          <Carousel>
+            <p>Slide1</p>
+            <p>Slide2</p>
+            <p>Slide3</p>
+          </Carousel>
+        );
+        expect(wrapper).toHaveState({ currentSlide: 0 });
+        map.keydown({ keyCode: 39 });
+        expect(wrapper).toHaveState({ currentSlide: 1 });
+      });
+
+      it('should not move to next slide when pressing right arrow key', () => {
```

Comment:
Include in the test description what prop is changing so we know why it shouldn't be moving to the next slide in this case